src/main/java/com/app/bookstore/service/shopping/cart/ShoppingCartService.java

```diff
@@ -0,0 +1,22 @@
+package com.app.bookstore.service.shopping.cart;
+
+import com.app.bookstore.dto.cart.item.PutCartItemRequestDto;
+import com.app.bookstore.dto.shopping.cart.ShoppingCartRequestDto;
+import com.app.bookstore.dto.shopping.cart.ShoppingCartResponseDto;
+import com.app.bookstore.model.User;
+import org.springframework.security.core.Authentication;
+
+public interface ShoppingCartService {
+    ShoppingCartResponseDto findShoppingCartByUserId(Long userId);

```

Comment:
Same as for CardItemService.
```suggestion
    ShoppingCartResponseDto findByUserId(Long userId);
```

Reply:
Make sure to simplify it everywhere.